cmd/cmd_renew.go

```diff
@@ -194,3 +202,25 @@ func merge(prevDomains []string, nextDomains []string) []string {
 	}
 	return prevDomains
 }
+
+func renewHook(ctx *cli.Context) error {
+	hook := ctx.String("renew-hook")
+	if hook == "" {
+		return nil
+	}
+
+	ctxCmd, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+
+	parts := strings.Split(hook, " ")
```

Comment:
Maybe best to use this instead?

```suggestion
	parts := strings.Fields(hook)
```

Reply:
Yes good suggestion!